go.mod

```diff
@@ -25,6 +25,8 @@ replace google.golang.org/grpc => google.golang.org/grpc v1.33.2
 
 replace github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.2-alpha.regen.4
 
+replace github.com/cosmos/cosmos-sdk => github.com/regen-network/cosmos-sdk v0.43.0-regen-1
```

Comment:
why do we need this?

Reply:
We added a small PR to our 0.43 fork https://github.com/regen-network/cosmos-sdk/pull/42